extensions/llama/python/llama.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any
+from aiconfig import Output, Prompt
+from aiconfig.Config import AIConfigRuntime
+from aiconfig.default_parsers.parameterized_model_parser import ParameterizedModelParser
+from aiconfig.model_parser import InferenceOptions
+from aiconfig.schema import ExecuteResult
+
+from llama_cpp import CreateCompletionResponse, Llama
+
+from aiconfig.util.params import resolve_prompt
+
+
+SUPPORTED_MODELS = ["llama-2-7b-chat"]
+
+
+class LlamaModelParser(ParameterizedModelParser):
+    def __init__(self, model_path: str) -> None:
+        super().__init__()
+        self.model_path = model_path
+
+    async def serialize(
+        self,
+        prompt_name: str,
+        data: Any,
+        ai_config: AIConfigRuntime,
+        parameters: dict | None = None,
+        **kwargs,
+    ) -> Prompt:
+        parameters = parameters or {}
+        out = Prompt(name="TODO", input="TODO")
```

Comment:
Implementation missing?

Reply:
yep, took a while just to get the hello world of this model parser :(